src/main/java/org/broadinstitute/hellbender/utils/nio/ReadsIterable.java

```diff
@@ -48,7 +48,7 @@ public ReadsIterator() throws IOException {
             // set high-level retries to deal with servers that might be temporarily overloaded
             // while we're reading a very long file from them.
             SeekableByteChannelPrefetcher chan = new SeekableByteChannelPrefetcher(
-                Files.newByteChannel(fpath, CloudStorageOptions.withChannelReopen(BucketUtils.NIO_MAX_REOPENS)), BUFSIZE);
+                Files.newByteChannel(fpath), BUFSIZE);
```

Comment:
Verify that the default reopens as set in `setGlobalNIODefaultOptions()` are applied here in the absence of passing it in explicitly.